.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -40,6 +40,17 @@ body:
     validations:
       required: true
 
+  - type: input
+    attributes:
+      label: Target device information (optional)
+      description: |
+        - Please provide the target device details if the issue occurs on or is related to Android/iOS.
```

Comment:
This could also be useful for reports about bugs on the web, since often the browser or OS can have an impact. Would you be able to expand the description to include this?